src/@utils/accessDetailsAndPricing.ts

```diff
@@ -291,56 +291,3 @@ export async function getAccessDetails(
     LoggerInstance.error('Error getting access details: ', error.message)
   }
 }
-
-export async function getAccessDetailsForAssets(
```

Comment:
cool, nice removal. This leaves many unused import at top of file, see the GitHub warnings further up

Reply:
Yeah good point. I've removed the unused imports and also there was a query that we weren't using anymore that's also removed